### DISCUSSION TITLE
GPS triggers foto/video

### DISCUSSION THREAD

**Discussion by TigerBiker** [gopro/labs#802] [General]
Hello, i am on tour with my motorcyle. Actually i use remotecontrol to start video and stop automaticly after 10 seconds. My wish is, that i can give some gps coordinates to my gopro and then she can start video or foto when we are at the defined coordinates. It should be possible, because gopro knows exactly where we are.  Can you make this possible? It would be good to give at least 20 coordinates at one time. greetings and have fun, thinking about my idea

**Comment by dnewman-gpsw** [comment]
Yes this is possible with Labs and GPS enabled cameras.

See https://gopro.github.io/labs/control/actions/

This is the key section:
```
c coordDistance - >cDistCMD then CMD, units in meters, compare distance from initial GPS location
c:X coord[A-Z]Distance - >c:XDistCMD There are up to 26 pre-stored GPS locations oMFRMA=latt,long thru oMFRMZ. If nothing is store in FRMx, it will initialize with the current location.
```

You can have up to 26 geofenced locations to cause triggers, although this is feature that has been limited testing.  One idea for this is you could define the path of a race track, and the camera should be scripted to only record while on the track, not in paddock.

**Reply by TigerBiker** [reply]
thank you very much for this information, it is exactly what i was looking for. 
I try now, to create a correct QR-Code. My opinion is, to store the coordinates of POIs along my track, e.g. castle, mountain, church, bridge, etc. My experience is, that is often very bussy on this places and it is better to concentrate to bike than to pick a picture.
